services/motion/builtin/move_request.go

```diff
@@ -777,12 +920,12 @@ func (mr *moveRequest) stop() error {
 	return nil
 }
 
-func toGeoPosePlanSteps(posesByComponent []map[resource.Name]spatialmath.Pose, geoPoses []spatialmath.GeoPose) ([]motion.PlanStep, error) {
+func toGeoPosePlanSteps(posesByComponent []motionplan.PlanStep, geoPoses []spatialmath.GeoPose) ([]motionplan.PlanStep, error) {
```

Comment:
Should this remain in this file? Thoughts on it being moved into `spatialmath/geo_obstacle.go`?

Reply:
I don't think it should be moved as if we were to move it that would cause spatialmath to depend on the motionplan package.